nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -589,18 +591,43 @@ public void testGCEnabled() {
   }
 
   @Test
-  public void testTableMetadataFilesCleanupDisable() throws NessieNotFoundException {
+  public void testGCEnabled() {
     Table icebergTable = catalog.loadTable(TABLE_IDENTIFIER);
+    icebergTable.updateProperties().set(TableProperties.GC_ENABLED, "true").commit();
+    Assertions.assertThat(icebergTable.properties().get(TableProperties.GC_ENABLED))
```

Comment:
`Assertions.assertThat(icebergTable.properties()).containsEntry(TableProperties.GC_ENABLED, "true");` is usually favorable, as it will print the content of the map if the check ever fails

Reply:
could you also please update the one in L581 to `Assertions.assertThat(icebergTable.properties())
        .containsEntry(TableProperties.GC_ENABLED, "false");`

Reply:
good point - fixed